File: ThaiTuanERP2025.Server/ThaiTuanERP2025.Application/Expense/Dtos/ExpensePaymentCommentAttachmentDto.cs
namespace ThaiTuanERP2025.Application.Expense.Dtos
{
	public sealed record ExpensePaymentCommentAttachmentDto(
		Guid Id,
		string FileName,
		string FileUrl,
		long FileSize,
		string? MimeType,
		Guid FileId
	);
	public sealed record CommentAttachmentRequest(
		string FileName,
		string FileUrl,
		long FileSize,
		string? MimeType,
		Guid? FileId
	);
}

File: ThaiTuanERP2025.Client/src/app/features/expense/models/expense-payment-comment-attachment.dto.ts
export interface ExpensePaymentCommentAttachment {
      fileName: string;
      fileUrl: string;
      fileSize: number;
      mimeType?: string;
}
export interface ExpensePaymentCommentAttachmentDto extends ExpensePaymentCommentAttachment {
      id: string;
}
export interface ExpensePaymentCommentAttachmentRequest extends ExpensePaymentCommentAttachment {
      fileId?: string;
}